styles/standard.json

```diff
@@ -269,6 +269,14 @@
 					"razed:railway":"rail",
 					"railway":"razed"
 				}
+			}
+		{
```

Comment:
indentation error

Reply:
I still can't figure that part out it has me confused

Reply:
Okay I managed to fix it